### DISCUSSION TITLE
Breaking Changes in Rename to React-JUCE

### DISCUSSION THREAD

**Discussion by nick-thompson** [nick-thompson/react-juce#207] [General]
Hey all,

I just wanted to highlight a quick heads up that #202 has just merged, which will be the first (and hopefully the worst) in a small series of breaking changes. I wanted to start up a discussion thread to alert any active contributors that this will cause some slight headaches for current work, but we'll restabilize very shortly. #34 is the issue documenting the reasons for the rename, if you want to see the context.

The only immediate change necessary for end users is that after updating to the latest `master`, you'll need to update your submodules again as `yoga` has been relocated from `blueprint/yoga` to `react_juce/yoga`, and you'll need to update your CMakeLists or Projucer projects to now refer to `react_juce` instead of `blueprint`. Hopefully that's an easy one, feel free to post here if you're having any issues and I'll help you out.

The forthcoming changes will be smaller:
* Renaming the top-level `blueprint::` namespace to `reactjuce::`
* Renaming javascript files which reference "blueprint."

cc @JoshMarler @olidacombe @stfufane @jrlanglois @eupston

**Comment by stfufane** [comment]
Great! Nothing broken here :)

**Comment by nick-thompson** [comment]
Hey folks, another update here, I just merged https://github.com/nick-thompson/blueprint/pull/216 for which the primary breaking change is the rename of the `blueprint::` namespace to `reactjuce::` and the rename of `BlueprintGenericEditor` to `GenericEditor`. Otherwise it should be a fairly smooth update.

I believe this is the last of the breakage here, and will be updating the docs to reflect the new name next. 🚀
